src/packaging/packages.targets

```diff
@@ -60,14 +60,37 @@
             <ILCompilerSdkFiles Include="bootstrappercpp" />
             <ILCompilerSdkFiles Include="System.Private.CoreLib.Native" Condition="'$(OsEnvironment)'!='Windows_NT'" />
 
+            <!-- ILCompiler.SDK Cpp Codegen support files -->
+            <ILCompilerSdkCppCodegenFiles Include="Native/gc/env/etmdummy.h" />
+            <ILCompilerSdkCppCodegenFiles Include="Native/gc/env/gcenv.base.h" />
```

Comment:
The headers from GC won't be needed in the SDK post my PR. They will be only needed to compile the runtime, not the auto-generated .cpp files.

(Eventually, we should not need even sal.h/gcenv.structs.h/gcenv.base.h in the SDK ... but there is a bit more cleanup required for that.)